pallets/aleph/src/migrations/v1_to_v2.rs

```diff
@@ -0,0 +1,38 @@
+use crate::Config;
+use frame_support::log;
+use frame_support::{
+    generate_storage_alias,
+    traits::{Get, GetStorageVersion, PalletInfoAccess, StorageVersion},
+    weights::Weight,
+};
+
+generate_storage_alias!(Aleph, SessionForValidatorsChange => Value<()>);
+generate_storage_alias!(Aleph, MillisecsPerBlock => Value<()>);
+generate_storage_alias!(Aleph, SessionPeriod => Value<()>);
+
+pub fn migrate<T: Config, P: GetStorageVersion + PalletInfoAccess>() -> Weight {
+    let on_chain_storage_version = <P as GetStorageVersion>::on_chain_storage_version();
+    let current_storage_version = <P as GetStorageVersion>::current_storage_version();
+
+    if on_chain_storage_version == 1 && current_storage_version == 2 {
+        log::info!(target: "pallet_aleph", "Running migration from STORAGE_VERSION 1 to 2");
+
+        SessionForValidatorsChange::kill();
+        MillisecsPerBlock::kill();
+        SessionPeriod::kill();
```

Comment:
I would love to see a sanity check here of whether the keys exist, before killing them. Because how else will we know if the migration actually does anything? (Unless `kill()` already does that?)

Reply:
If there is no storage then kill does nothing.

Reply:
Ok, so I think we really need some logs here that can make us confident that the migration went **EXACTLY AS EXPECTED**, i.e. that every key that we expected was present etc. This is mainly for the purpose of testing -- if we leave it as it is then we might be killing non-existing keys and stuff like that.

Reply:
So simple check if all of them exist or reading values as well? I don't see how they cannot exit but logging doesn't hurt.

Reply:
They can not exists because we might be doing the migration wrong, which is exactly what happened in one of the previous PRs that a few people (including me btw.) read, tested and accepted.

Reply:
Added logs.